### PR TITLE
Added sample version of server managing client data

### PR DESCRIPTION
**Pull request by katevi**
Implemented this tutorial: https://spring.io/guides/gs/messaging-stomp-websocket/ ; configurated server to listen multiple ports with this arcticle: https://tech.asimio.net/2016/12/15/Configuring-Tomcat-to-Listen-on-Multiple-ports-using-Spring-Boot.html

### CODE REVIEW

File: chat-sample/src/main/java/com/example/messagingstompwebsocket/MessageController.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.web.util.HtmlUtils;
+
+@Controller
+public class MessageController {
+
+
+    @MessageMapping("/hello")
+    @SendTo("/topic/greetings")
+    public Message greeting(TextMessage message) throws Exception {
+        Thread.sleep(1000); // simulated delay
```
Comment (AndreyKoltsov1997) [inline chat-sample/src/main/java/com/example/messagingstompwebsocket/MessageController.java]: Why would you simulate the delay?

File: chat-sample/src/main/java/com/example/messagingstompwebsocket/config/EmbeddedTomcatConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+@Configuration
+public class EmbeddedTomcatConfiguration {
+
+    @Value("${server.port}")
+    private String serverPort;
+
+    @Value("${management.port:${server.port}}")
+    private String managementPort;
+
+    @Value("${server.additionalPorts:null}")
```
Comment (AndreyKoltsov1997) [inline chat-sample/src/main/java/com/example/messagingstompwebsocket/config/EmbeddedTomcatConfiguration.java]: What's the purpose of null value here?

File: chat-sample/src/main/java/com/example/messagingstompwebsocket/config/EmbeddedTomcatConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            tomcat.addAdditionalTomcatConnectors(additionalConnectors);
+        }
+        return tomcat;
+    }
+
+    private Connector[] additionalConnector() {
+        if (StringUtils.isBlank(this.additionalPorts)) {
+            return null;
+        }
+        Set<String> defaultPorts = Sets.newHashSet(this.serverPort, this.managementPort);
```
Comment (AndreyKoltsov1997) [inline chat-sample/src/main/java/com/example/messagingstompwebsocket/config/EmbeddedTomcatConfiguration.java]: Could it be `final`?

File: chat-sample/src/main/resources/static/jquery-3.4.1.min.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
...tion")),ot.type="checkbox",y.checkOn=""!==ot.value,y.optSelected=at.selected,(ot=E.createElement("input")).value="t",ot.type="radio",y.radioValue="t"===ot.value;var ht,gt=k.expr.attrHandle;k.fn.extend({attr:function(e,t){return _(this,k.attr,e,t,1<arguments.length)},removeAttr:function(e){return this.each(function(){k.removeAttr(this,e)})}}),k.extend({attr:function(e,t,n){var r,i,o=e.nodeType;if(3!==o&&8!==o&&2!==o)return"undefined"==typeof e.getAttribute?k.prop(e,t,n):(1===o&&k.isXMLDoc(e)||(i=k.attrHooks[t.toLowerCase()]||(k.expr.match.bool.test(t)?ht:void 0)),void 0!==n?null===n?void k.removeAttr(e,t):i&&"set"in i&&void 0!==(r=i.set(e,n,t))?r:(e.setAttribute(t,n+""),n):i&&"get"in i&&null!==(r=i.get(e,t))?r:null==(r=k.find.attr(e,t))?void 0:r)},attrHooks:{type:{set:function(e,t){if(!y.radioValue&&"radio"===t&&A(e,"input")){var n=e.value;return e.setAttribute("type",t),n&&(e.value=n),t}}}},removeAttr:function(e,t){var n,r=0,i=t&&t.match(R);if(i&&1===e.nodeType)while(n=i[r++])e.removeAttribute(n)}}),ht={set:function(e,t,n){return!1===t?k.removeAttr(e,n):e.setAttribute(n,n),n}},k.each(k.expr.match.bool.source.match(/\w+/g),function(e,t){var a=gt[t]||k.find.attr;gt[t]=function(e,t,n)
```
Comment (AndreyKoltsov1997) [inline chat-sample/src/main/resources/static/jquery-3.4.1.min.js]: I don't think source code is the best way to store compressed JS libraries. Please, consider using a package manager or Gradle to load them.

**Comment by katevi** [comment]
This draft was was redone in the following PR

**Pull request closed without merging**